e2e/msa/msaTokens.test.ts

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable mocha/no-skipped-tests */
+import '@frequency-chain/api-augment';
+import assert from 'assert';
+import { ExtrinsicHelper } from '../scaffolding/extrinsicHelpers';
+import { HexString } from '@polkadot/util/types';
+import { isDev, isTestnet } from '../scaffolding/env';
+
+const msaId = 1234; // Example MSA ID for testing
+let checksummedEthAddress: HexString;
+
+if (isTestnet()) {
+  checksummedEthAddress = '0x05500A07f5fD359e9E785c74E21d5b180e63B63b'; // Example checksummed Ethereum address for MSA ID 1234 on Paseo testnet
+} else {
+  checksummedEthAddress = '0xF5b82ff246a2F4226749bd78B1bDaE28Cfffb9f7'; // Example checksummed Ethereum address for MSA ID 1234 on a development chain
+}
+
+describe('MSAs Holding Tokens', function () {
+  describe('getEthereumAddressForMsaId', function () {
```

Comment:
While not quite an e2e test (and perhaps it is coming), but it would be a good idea to have a test that shows an external js library generating the same value so that we make sure that a external implementation of the library generates the same result as the chain. Also as an example of how to use it to generate the value.

Reply:
I'll add JS functions to the scaffolding that implements both `CREATE2` and our MSA address derivation logic.